cartography/intel/gcp/loadbalancer.py

```diff
@@ -463,6 +463,7 @@ def transfrom_global_url_maps(url_maps: List, project_id: str):
     for url_map in url_maps:
         url_map['region'] = 'global'
         url_map['type'] = 'global'
+        url_map['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='load_balancer')
```

Comment:
this resource_name does not exist in cloudconsolelinks project

Reply:
As commented above